.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,75 @@
+name: Publish Release
+
+# changed from pull_request to push
+  # added master branch
+
+# replaced merged-true bin/filter for filter-by-commit (name tbd) to check if #(1-9) is available in commit: this ensures the workflow is run when a merge happens.
+
+# took out refs because we're no longer running this from pr but from the branch itself so we should be good.
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  filter:
+    name: Filter for Merges
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    # stage for adding to thefrontside/actions
+    - uses: taras/georgia/.github/actions/filter-by-commit@release-x
+
+  tests:
```

Comment:
Now I'm wondering if we should be blocking the publish on tests because this PR should not have been merged to the branch without passing the tests. Until we're running our CI in GitHub, I'm thinking it's best not to actually run the tests since there will be invariable divergence between the Circle environment and the GitHub environment.

Reply:
we can take the tests out and put it back in when it's more appropriate.

oh and you can track my progress [here](https://github.com/minkimcello/microstates.js) @ release-0.

Reply:
and actions [here](https://github.com/minkimcello/actions)